internal/services/integrationtesting/testconfigs/self.yaml

```diff
@@ -0,0 +1,23 @@
+---
+schema: >-
+  definition user {
+    relation related: user
+    permission me_or_related = related + self
```

Comment:
can we add to this example uses of `self#` and `self->`?

Reply:
This PR doesn't support that currently. If we want to support it, it can be done, but it is much more complicated. I'd prefer to only add support for it if necessary